modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -68,6 +68,9 @@ def to_internal
       if separation_pay_received_date?
         form_data['servicePay']['separationPay']['receivedDate'] = transform_separation_pay_received_date
       end
+      if invalid_change_of_address_ending_date?
```

Comment:
I always struggle with this because I don't like how it reads as if it is going to always transform something.
Because in most cases, it probably won't need to be transformed.
Having less code is nice. I just get uncomfortable with how it reads.  
I have no strong objections to making that change, but do you kind of see what I mean?

Reply:
yeah that's a good point. in that case, maybe just make it an inline `if` like line 76, i.e.:
```ruby
form_data['veteran']['changeOfAddress'] = transform_change_of_address_ending_date if invalid_change_of_address_ending_date?
```

Reply:
Haha, I wish I could, but that makes that line too long.  😂 
(`Layout/LineLength: Line is too long. [129/120]`)